### ISSUE TITLE
Mute All - button does not work 

### ISSUE THREAD

**Issue by mMARUF** [mozilla/hubs#5683]
**OS:**
macOS Monterey version: 12.5.1

**Browser:**
Chrome Version 104.0.5112.101

**Steps to Reproduce:**
1) Go to https://hubs.mozilla.com/
2) Inside a room Press on the button **Mute All** 

**Actual Result:**
**Mute All** button is not working. The users inside the rooms are not muted **(Check the attached GIF)**

**Expected Result:**
After pressing **Mute All** button, the users inside the room should be muted.

**Attachment:**
![ezgif com-gif-maker](https://user-images.githubusercontent.com/45368054/187289318-609b0010-582b-4f8d-a6f3-1844bea335ce.gif)

**Comment by matthewbcool** [comment]
Thanks for the report. Notice those participants in the capture are in the lobby. 

Because they are not in-room, they have will not be able to use their mic and so there is no need to mute them. I can see how the expected behavior may be to change the state of the ui regardless though. Mute All still functions as expected in-room, I'll change the title of this issue to reflect this.

**Comment by emclaren** [comment]
Yes, this would be great for us to update in our interface to make it more clear. This isn't on our immediate roadmap, but flagging this for us to explore as we polish up the client experience

**Comment by mMARUF** [comment]
Updated the bug description, when we try to press **Mute All** 2nd time, the system does not unmutes the users. And also After pressing **Mute All**, a color change of the button will be helpful.
